pandas/io/formats/style_render.py

```diff
@@ -477,6 +489,109 @@ def _generate_col_header_row(self, iter: tuple, max_cols: int, col_lengths: dict
 
         return index_blanks + column_name + column_headers
 
+    def _generate_descriptor_row(self, iter: tuple, max_cols: int):
+        """
+        Generate the row containing calculated descriptor values for columns:
+
+         +----------------------------+---------------+---------------------------+
+         |  index_blanks ...          | descriptor_i  |  value_i by col           |
+         +----------------------------+---------------+---------------------------+
+
+        Parameters
+        ----------
+        iter : tuple
+            Looping variables from outer scope
+        max_cols : int
+            Permissible number of columns
+
+        Returns
+        -------
+        list of elements
+        """
+
+        r, descriptor = iter
+
+        # number of index blanks is governed by number of hidden index levels
+        index_blanks = [
+            _element("th", self.css["blank"], self.css["blank_value"], True)
+        ] * (self.index.nlevels - sum(self.hide_index_) - 1)
+
+        if isinstance(descriptor, str):
+            name: str | None = descriptor
+            func: Callable = getattr(Series, descriptor)
+        elif isinstance(descriptor, tuple):
+            name, func = descriptor[0], descriptor[1]
+        else:
+            name, func = None, descriptor
```

Comment:
When the descriptor is a `Callable` I think it would be best to try `getattr(descriptor, "name", None)` for the `name` rather than committing to a blank name. For example `pd.Series.mean.__name__` is `mean`.

Reply:
- [x] chged. note the case of `"<lambda>"` i thought was worth blanking